linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_pam_enabled/oval/shared.xml

```diff
@@ -0,0 +1,23 @@
+<def-group oval_version="5.11">
+  <definition class="compliance" id="smartcard_pam_enabled" version="3">
+   {{{ oval_metadata("Enable Smart Card logins using PAM", title="Enable Smart Card Login in PAM", affected_platforms=["multi_platform_sle"]) }}}
```

Comment:
you may remove the `affected_platforms` here... it's automatically picked up by the build system from the rule `prodtype` field. The title is also retrieved from the rule `title` field.
```suggestion
   {{{ oval_metadata("Enable Smart Card logins using PAM") }}}
```

Reply:
will do